content/posts/newsletter-017/index.md

```diff
@@ -85,6 +85,35 @@ This month's updates:
 [wor-discord]: https://discord.gg/JGeVt5XwPP
 [wor-website]: https://anthropicstudios.com/way-of-rhea
 
+### [Vollmond][vollmond-itch]
+
+![Vollmond preview](vollmond-preview.png)
+
+[Vollmond][vollmond-itch] a small non violent adventure in the style of Zelda 2.
```

Comment:
```suggestion
[Vollmond][vollmond-itch] is a small non-violent adventure in the style of Zelda 2.
```

Reply:
I agree. Do you want me to change it or do you?

Reply:
I've applied the change :+1: